script/test/test.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/sh
 
 # Run linter (checks code style)
-pycodestyle  --select E src/ test/ --exclude src/filters,src/parser/evaql
-
+flake8 --select E,F src/ test/ --exclude src/filters,src/parser/evaql
+exit_code=$?
 # Run unit tests
 PYTHONPATH=./ pytest test/ --cov-report= --cov-config=.coveragerc --cov=src/ -s -v --log-level=10
 
+if [ $exit_code -ne 0 ]; 
```

Comment:
It might be helpful to add a comment stating that this is important for `travis`.

Reply:
Done